plugins/pulp_rpm/plugins/importers/yum/parse/treeinfo.py

```diff
@@ -92,7 +92,8 @@ def sync(sync_conduit, feed, working_dir, nectar_config, report, progress_callba
         else:
             _LOGGER.error('some distro file downloads failed')
             report['state'] = constants.STATE_FAILED
-            report['error_details'] = [(fail.url, fail.error_report) for fail in listener.failed_reports]
+            report['error_details'] = [(fail.url, {'error_message': fail.error_msg})
+                                       for fail in listener.failed_reports]
```

Comment:
This seems like it will be an API change. Is that correct? If so, we shouldn't do this until 3.0 IMO.

Also, this PR needs tests if we are going to go through with it.